CppWinRTProjectionSample/SimpleMathProjection/SimpleMathProjection.nuspec

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+	<metadata>
+		<id>SimpleMathComponent</id>
+		<version>0.1.0-prerelease</version>
+		<authors>Contoso Math Inc.</authors>
+		<description>A simple component with basic math operations</description>
+		<dependencies>
+			<group targetFramework="net5.0">
+				<dependency id="Microsoft.Windows.CsWinRT" version="0.1.0-prerelease.200922.1" exclude="Build,Analyzers" />
```

Comment:
consider using a local nuget source (from the build output) and setting version to a variable

Reply:
Neither @angelazhangmsft nor I are nuget/msbuild experts - can you make a PR on how to do this?

Reply:
I'm okay with taking an IOU on this, completing the PR